RainbowForge/Dump/DumpHelper.cs

```diff
@@ -104,6 +104,26 @@ public static void DumpBin(string filename, Stream stream, long writeOffset = 0,
 				stream.CopyTo(fs);
 			else
 				stream.CopyStream(fs, writeLength);
+			fs.Dispose();
+			stream.Dispose();
+		}
+
+		public static void DumpWem(string filename, Stream stream, long writeOffset = 0, int writeLength = -1)
+		{
+			MemoryStream ms = new MemoryStream();
+			stream.CopyTo(ms);
+			int RiffOffset = System.Text.Encoding.Latin1.GetString(ms.ToArray()).IndexOf("RIFF");
```

Comment:
Can you do a linear search through the stream to find the offset of the RIFF header? converting the entire stream to a string is super inefficient and might cause issues with null terminators in some cases